prep-javascript-variables/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const fullName = 'AndrewOrtiz';
+      console.log(fullName);
+      console.log(typeof fullName);
+      const numberOfPets = '1';
+      console.log(numberOfPets);
+      console.log(typeof 1);
+      const likesVegetables = 'true';
+      Boolean(true);
+      console.log(likesVegetables);
+      console.log(typeof true);
+      const nothing = 'null';
+      console.log(nothing);
+      console.log(typeof null);
+      const never = 'undefined';
```

Comment:
here you want to use a `let` variable because you cannot declare a variable with const and not assign it a value. In this case you assigned a string named undefined to it. make sure to update this